src/test_utils.rs

```diff
@@ -39,6 +40,47 @@ pub const AVATAR_900x900_BYTES: &[u8] = include_bytes!("../test-data/image/avata
 static CONTEXT_NAMES: Lazy<std::sync::RwLock<BTreeMap<u32, String>>> =
     Lazy::new(|| std::sync::RwLock::new(BTreeMap::new()));
 
+pub struct AcManager {
+    log_tx: Sender<Event>,
+    _log_sink: LogSink,
+    accounts: Vec<Rc<TestContext>>,
+}
+
+impl AcManager {
+    pub async fn new() -> Self {
+        let (log_tx, _log_sink) = LogSink::create();
+        Self {
+            log_tx,
+            _log_sink,
+            accounts: vec![],
+        }
+    }
+
+    pub async fn ac_alice(&mut self) -> Rc<TestContext> {
```

Comment:
if we're renaming away from `AcManager` these should be renamed too, maybe simply `.alice()` is fine here